cmd/tuf/app/sign.go

```diff
@@ -136,7 +137,15 @@ func getSigner(ctx context.Context, sk bool, keyRef string) (signature.Signer, e
 		return pivKey.SignerVerifier()
 	}
 	// A key reference was provided.
-	return csignature.SignerVerifierFromKeyRef(ctx, keyRef, nil)
+	signer, err := signature.LoadSignerFromPEMFile(keyRef, crypto.SHA256, nil)
+	if err != nil {
```

Comment:
If I'm reading this correctly, when a user is trying to use a PEM key file and there's an error in that path, they will never see what the error message is. Is that correct? Should we separate out online and offline key usage?

Reply:
Good catch @joshuagl I just copy pasted the code from the `tests/utils.go`. Let me update.

Reply:
The problem with separating them as the code is now, is because `csignature.SignerVerifierFromKeyRef` also loads an online key: https://github.com/sigstore/cosign/blob/f7313145443852330736715821faf2ebba645f3d/pkg/signature/keys.go#L104
Specifically`-----BEGIN ENCRYPTED COSIGN PRIVATE KEY-----` keys. So we would need to do some introspection first, my recommendation would be to make sure both error messages are printed with clear information on how the keys where attempted to be loaded.

Reply:
Let's add a comment? 
I'd hope honestly we can propogate the logic back to sigstore/sigstore actually (to remove the restriction on COSIGN key headers in `SignerVerifierFromKeyRef`)